shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala

```diff
@@ -756,6 +766,20 @@ object GlutenConfig {
       .booleanConf
       .createWithDefault(false)
 
+  val COLUMNAR_VELOX_CONNECTOR_IO_THREADS =
+    buildConf("spark.gluten.sql.columnar.backend.velox.IOThreads")
+      .internal()
+      .doc("The IO threads for cpnnector split preloading")
+      .intConf
+      .createWithDefault(Runtime.getRuntime.availableProcessors)
```

Comment:
using all threads by default maybe too aggressive, how about set this to 1?

Reply:
synced with yuan offline, set it as 0 to disable this feature by default.